YardSaleDB/2 data.sql

```diff
@@ -0,0 +1,15 @@
+use YardSaleDB
+go 
+delete Item
+go
+
+insert Item(ItemName, [Description], Category, Donor, DateReceived, DateSold, Price)
+select 'Tricycle', 'Red and blue; heavily used but still functional', 'Recreation', 'Tom Smith', '11/27/2022', null, 15
+union select 'Scarf', 'Women''s; hardly worn', 'Clothing', 'Jane Jones', '05/22/2022', '06/14/2022', 5
+union select 'Dresser', '5-drawer; slightly shaky but very good condition', 'Home', 'Jane Jones', '01/04/2023', null, 35
+union select 'Guitar', 'missing string; small', 'Recreation', 'Sam Williams', '07/18/2022', '07/25/2022', 0
```

Comment:
"0" is not an option - the spec specifically said to write 'free' for such instances...